Update environment creation document [do not merge until MGuoynes confirms changes complete] (#250)
launchdarkly/LaunchDarkly-Docs: src/content/topics/home/organizing-flags/environments.mdx

![The "Switch environment" menu.](env-switcher.png)

You can manage your environments from the **Account settings** page. Here, you can add new environments to a project, for example, to give each developer on your team their own environment for local testing.

To add a new environment:

1. Navigate to the **Account settings** page.
2. Click the **Projects** tab. Find the project to which you want to add a new environment.

> 3. Navigate to the selected project by clicking on the **Project** link.

Reviewer: ```suggestion
3. Navigate to the selected project by clicking the name of the project.
```

1. Navigate to the the **Account settings** page.
2. Click the **Projects** tab.

> 3. In the **Projects** results click on a project name to navigate to that project.

Reviewer: ```suggestion
3. Navigate to the selected project by clicking the name of the project.
```
If I'm following correctly, this step is the same as what you're doing in environments.mdx/line 36. Is that right? If so, let's use the same language.

4. Click the **overflow menu** next to your environment.

> 5. Select "Edit environment." An "Edit environment" panel appears.

Reviewer 2: ```suggestion
5. Select "Edit environment." The "Edit environment" panel appears.
```

If the TTL is set to zero minutes, the SDK will not use a local cache, and every call to `variation` will make a remote request to our CDN. You can set your TTL to zero in testing environments to see changes reflected immediately, but we do not recommend a zero minute TTL in production.

In high volume PHP environments, we strongly recommend using our Relay Proxy. To learn more, read [The Relay Proxy](/home/relay-proxy).

To set the TTL:

1. Navigate to the the **Account settings** page.
2. Click the **Projects** tab.

> 3. In the **Projects** results click on a project name to navigate to that project.

Reviewer: ```suggestion
3. Navigate to the selected project by clicking the name of the project.
```
Same comment as above, I think we can standardize the language here.

4. Click the **overflow menu** next to your environment.

> 5. Select "Edit environment." An "Edit environment" panel appears.

Reviewer 2: ```suggestion
5. Select "Edit environment." The "Edit environment" panel appears.
```

> 6. Enter a value in the **TTL input** and once done click **Save Environment**.

Reviewer: ```suggestion
6. Enter a value in the **TTL input** and click **Save environment**.
```
